trimmer_decorator.rb

```diff
@@ -2,7 +2,6 @@
 
 class TrimmerDecorator < BaseDecorator
   def correct_name
-    trimmed_name = @nameable.correct_name[0..9]
-    trimmed_name.length == 10 ? trimmed_name : @nameable.correct_name
+    @nameable.correct_name[0..9]
```

Comment:
- [ ] Great job making an effort to fix the requested change. Unfortunately, you have instead deleted the line of code instead of correcting it from == to > . You can refactor to look like the example snippet below:
```
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```